build/gulpfile.js

```diff
@@ -56,12 +58,17 @@ module.exports = function(gulp, config, pluginName){
         module: 'commonjs',
         declarationFiles: true,
         noExternalResolve: false,
-        removeComments: true
-      })))
-      .on('error', plugins.notify.onError({
```

Comment:
ah, i don't have such notifications supported on my system. i can put that back, i just liked the output of the reported (in console) more.